src/test/java/core/basesyntax/CalculatorTest.java

```diff
@@ -0,0 +1,260 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+public class CalculatorTest {
+    private static final char ADDITION = '+';
+    private static final char SUBTRACTION = '-';
+    private static final char DIVISION = '/';
+    private static final char MULTIPLICATION = '*';
+    private static final char RAISING_TO_A_POWER = '^';
+    private static final char ILLEGAL_OPERATION = '!';
+    private static final double POSITIVE_VALUE = 9876.143;
+    private static final double NEGATIVE_VALUE = -1234.167;
+    private static final double ZERO = 0;
+    private static final double DELTA = 0.0001;
+    private static Calculate calculator;
+    private double expected;
+    private double actual;
+
+    @BeforeAll
+    static void beforeAll() {
+        calculator = new Calculator();
+    }
+
+    @Test
+    void checkAdditionTwoPositiveOperands_Ok() {
```

Comment:
```suggestion
    void calculate_addTwoPositiveOperandsOk() {
```
that's common naming convention for methods, it's better to use google java code style guides